README.md

```diff
@@ -5,13 +5,13 @@
 <th><b><a href="">Doxygen Documentation (TODO)</a></b></th>
 </tr></table>
 
-`cuCollections` (`cuco`) is an open-source, header-only library of GPU-accelerated, concurrent data structures. 
+`cuCollections` (`cuco`) is an open-source, header-only library of GPU-accelerated, concurrent data structures.
```

Comment:
No clue why the README had so many lines with trailing whitespaces.

Reply:
Simply because no linter was used when editing the file. Thanks for the fix.